modules/virt-importing-vm-cli.adoc

```diff
@@ -1,6 +1,8 @@
 // Module included in the following assemblies:
+//
 // * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
-[id='virt-importing-vm-cli_{context}']
+
+[id="virt-importing-vm-cli_{context}"]
```

Comment:
Seriously good catches on these module IDs. This is the kind of thing I miss when doing reviews. 😱